index.js

```diff
@@ -0,0 +1,223 @@
+import keys from './js/keys_data.js';
+
+class KeyboardClass {
+    constructor() {
+        this.enLang = localStorage.getItem('lang') || 'en';
+        this.shift = false;
+        this.caps = false;
+        this.keys = keys;
+    }
+
+    generateBaseStructure() {
+        const body = document.querySelector('body');
+        this.display = document.createElement('div');
+        this.display.classList.add('display');
+        this.wrapper = document.createElement('div');
+        this.wrapper.classList.add('wrapper');
+        this.textarea = document.createElement('textarea');
+        this.textarea.classList.add('display__area');
+        this.wrapper.appendChild(this.textarea);
+        this.display.appendChild(this.wrapper);
+        body.appendChild(this.display);
+        this.keyboard = document.createElement('div');
+        this.keyboard.classList.add('keyboard');
+        this.wrapperKeyboard = document.createElement('div');
+        this.wrapperKeyboard.classList.add('wrapper');
+        this.holder = document.createElement('div');
+        this.holder.classList.add('keyboard__keys-holder');
+        this.wrapperKeyboard.appendChild(this.holder);
+        this.keyboard.appendChild(this.wrapperKeyboard);
+        body.appendChild(this.keyboard);
+        this.about = document.createElement('div');
+        this.about.classList.add('about-keyboard');
+        this.wrapperAbout = document.createElement('div');
+        this.wrapperAbout.classList.add('wrapper');
+        this.wrapperAbout.classList.add('about-keyboard__wrapper');
+        this.wrapperAbout.innerHTML = '<p class="about-keyboard__text">Change language command: AltLeft + ControlLeft | RU/ENG</p><p class="about-keyboard__text">OS: Windows</p><p class="about-keyboard__text">Developer: ptatarsk</p>';
+        this.about.appendChild(this.wrapperAbout);
+        body.appendChild(this.about);
+    }
+
+    createKeys() {
+        this.holder.querySelectorAll('div').forEach((el) => {
+            el.remove();
+        });
+        if (this.enLang === 'en' && this.caps === false && this.shift === false) {
+            this.value = this.keys.keysEn;
+        } else if (this.enLang === 'en' && (this.caps === true || this.shift === true)) {
+            this.value = this.keys.keysEnCaps;
+        } else if (this.enLang === 'ru' && this.caps === false && this.shift === false) {
```

Comment:
Magic values. Encapsulate "this.enLang === 'ru' && this.caps === false && this.shift === false" into method -> "isDesiredLang"